RecoLocalTracker/SiPixelRecHits/interface/PixelCPEFastAlpaka.h

```diff
@@ -0,0 +1,51 @@
+#ifndef RecoLocalTracker_SiPixelRecHits_PixelCPEFastAlpaka_h
+#define RecoLocalTracker_SiPixelRecHits_PixelCPEFastAlpaka_h
+
+#include <utility>
+
+#include "CondFormats/SiPixelTransient/interface/SiPixelGenError.h"
+#include "CondFormats/SiPixelTransient/interface/SiPixelTemplate.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEFastParams.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEGenericBase.h"
+// #include "RecoLocalTracker/SiPixelRecHits/interface/alpaka/pixelCPEforGPU.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+
+class MagneticField;
+template <typename TrackerTraits>
+class PixelCPEFastAlpaka final : public PixelCPEGenericBase {
```

Comment:
It looks like this is an EventSetup data product, but I didn't see any ESProducer for it.